dogs-initial/src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+
+        <!--*** Part 1-->
+
+<configuration>
+<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+        <pattern>[%d{yyyy-MM-dd' 'HH:mm:ss.SSS}] [%25.25class] [%thread] [%line] [%-5level] %m%n</pattern>
+    </encoder>
+</appender>
+
+<appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <File>D:\Users\Github Repos\java-dogs-logging\var\tmp\mylog.log</File>
+    <append>true</append>
+    <encoder>
+        <pattern>[%d{yyyy-MM-dd' 'HH:mm:ss.SSS}] [%C] [%t] [%L] [%-5p] %m%n</pattern>
+    </encoder>
+
+    <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+
+        <!-- rollover daily -->
+        <fileNamePattern>D:\Users\Github Repos\java-dogs-logging\var\tmp\mylog.%d{yyyy-MM-dd HH}.%i.txt</fileNamePattern>
+        <maxFileSize>15MB</maxFileSize>
+
+        <!-- keep 30 days' worth of history -->
+        <maxHistory>30</maxHistory>
+        <totalSizeCap>5GB</totalSizeCap>
+    </rollingPolicy>
+</appender>
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT"/>
+    </root>
+
+<!--*** Part 3 below -->
+
+<!--*** Part 2-->
+
+    <root level="INFO">
+        <appender-ref ref="FILE"/>
+        <appender-ref ref="STDOUT"/>
+    </root>
+
+
+
+<!--*** Part 4-->
+
+<appender name="TOMCAT" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <File>/var/tmp/myTomcatLog.log</File>
+    <append>true</append>
+    <encoder>
+        <pattern>[%d{yyyy-MM-dd'T'HH:mm:ss.sss'Z'}] [%C] [%t] [%L] [%-5p] %m%n</pattern>
```

Comment:
Line 49 - You might have forgotten to modify your file path to be the same as your other log file's destination path (going to the `D` drive).